### PR TITLE
Make custom status handling in NoILS driver more robust.

### PR DESCRIPTION
**Pull request by EreMaijala**

### CODE REVIEW

File: module/VuFind/src/VuFind/ILS/Driver/NoILS.php [hunk 1]
```diff
@@ -140,22 +140,17 @@ public function getStatus($id)
     {
         $useStatus = $this->config['settings']['useStatus'] ?? 'none';
         if ($useStatus == 'custom') {
-            $status = $this->translate($this->config['Status']['status']);
+            $status = $this->translate($this->config['Status']['status'] ?? '');
             return [
                 [
                     'id' => $id,
-                    'availability' => $this->config['Status']['availability'],
+                    'availability' => (bool)($this->config['Status']['availability'] ?? false),
```
Comment (demiankatz) [inline module/VuFind/src/VuFind/ILS/Driver/NoILS.php]: Should you avoid casting this to bool in order to support the integer constants (and for consistency with getHolding below)?

**Review by demiankatz — CHANGES_REQUESTED** [review]
This makes sense to me -- just one minor question.

File: module/VuFind/src/VuFind/ILS/Driver/NoILS.php (hunk 1 above)
Comment (EreMaijala) [reply module/VuFind/src/VuFind/ILS/Driver/NoILS.php]: Oops, of course. Fixed.

**Review by demiankatz — APPROVED** [review]
Thanks, @EreMaijala!

**Pull request merged**